scripts/content-repo-preview/build.ts

```diff
@@ -1,12 +1,20 @@
 import { execFileSync } from 'child_process'
 import path from 'path'
 import fs from 'fs'
+import { IoPreviewBuilder } from './builders/io-sites'
+import { DeveloperPreviewBuilder } from './builders/developer'
+
+/**
+ * Controls how the application is built, using either the `IoPreviewBuilder` or `DeveloperPreviewBuilder`
+ */
+type PreviewMode = 'io' | 'developer'
+
+// our CWD
+const cwd = process.cwd()
 
 function checkEnvVars() {
 	// Filter out defined env vars, leaving only the missing ones
-	const missingEnvVars = ['REPO', 'DEV_IO', 'IS_CONTENT_PREVIEW'].filter(
-		(key) => !process.env[key]
-	)
+	const missingEnvVars = ['REPO'].filter((key) => !process.env[key])
```

Comment:
This doesn't really make a ton of sense now given that we are deriving most of the environment variables 💭